src/cogs/help.py

```diff
@@ -0,0 +1,69 @@
+from typing import Optional
+
+from discord.ext import commands
+from discord.utils import get
+from helpers.helper_funcs import create_embed
+
+
+def syntax(client, command):
+    cmd_and_aliases = client.command_prefix[0] + "|".join([str(command), *command.aliases])
+    params = []
+    for key, value in command.params.items():
+        if key not in ("self", "context"):
+            params.append(f"[{key}]" if "NoneType" in str(value) else f"<{key}>")
+    params = " ".join(params)
+    return f"```{cmd_and_aliases} {params}```"
+
+
+class Help(commands.Cog, name="help", description="Get info about the bot and how to use it"):
+    def __init__(self, client):
+        self.client = client
+        self.client.remove_command("help")
+        self.cogs = [c for c in self.client.cogs.keys()]
+
+    async def help(self, context):
+        fields = [[f"All `{cog}` Commands:", "".join([f" • `{cmd}`\n" for cmd in self.client.get_cog(cog).get_commands()]).rstrip()] for cog in self.cogs]
+        embed = create_embed(
+            "John \"Not a robot\" Peter: Help",
```

Comment:
Would it be possible for this to pull from the bots current name? Sometimes we change the name on occasion (this might get more frequent if #19 merged)